src/components/index.ts

```diff
@@ -1,34 +1,38 @@
-import { AnchorListItem } from './anchor-list-item/anchor-list-item';
-import { ColorDescription } from './color-description/color-description';
-import { ComponentName } from './component-name/component-name';
-import { ComponentRules } from './component-rules/component-rules';
-import { DocFooter } from './doc-footer/doc-footer';
-import { Frame } from './frame/frame';
-import { FunctionArgument } from './function-arguments/function-argument';
-import { FunctionArguments } from './function-arguments/function-arguments';
-import { InformationBox } from './information-box/information-box';
-import { InformationBoxTitle } from './information-box-title/information-box-title';
-import { Link } from './link/link';
-import { LinkComponent } from './link-component/link-component';
-import { MultipleStoryElementsWrapper } from './multiple-story-elements-wrapper/multiple-story-elements-wrapper';
-import { Paragraph } from './paragraph/paragraph';
-import { RelatedComponent } from './related-component/related-component';
-import { RelatedComponents } from './related-components/related-components';
-import { SectionName } from './section-name/section-name';
-import { StoryDescription } from './story-description/story-description';
-// import { Tip } from "./tip/tip"; // uses attention box - not exported for now
-import { Title } from './title/title';
-import { TokenTable } from './token-table/token-table';
-import { UnstyledList } from './unstyled-list/unstyled-list';
-import { UnstyledListItem } from './unstyled-list-item/unstyled-list-item';
-import { UsageGuidelines } from './usage-guidelines/usage-guidelines';
-import { VisualDescription } from './visual-description/visual-description';
+import { default as AnchorListItem } from './anchor-list-item/anchor-list-item';
```

Comment:
I think you can export them already
```suggestion
export { default as AnchorListItem } from './anchor-list-item/anchor-list-item';
```